### DISCUSSION TITLE
GBuffer: support half or quater sized texture

### DISCUSSION THREAD

**Discussion by WangHoi** [DiligentGraphics/DiligentFX#245] [Ideas]
Will it be a good idea, that `GBuffer` can manage texture with size proportion to full resolution?

So `GBuffer::Resize` can resize with the size ratio.

**Comment by TheMostDiligent** [comment]
I don't see a need for this. G-buffer is unaware of whether it is attached to a screen, used for off-screen rendering with some predefined size or used for any other purpose.
If you need to scale the screen resolution, it can be easily done outside of G-buffer.

**Comment by WangHoi** [comment]
OK, thanks for your reply.
